Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -278,6 +284,11 @@ - (NSArray *)accessibilityElements
 
 @implementation ASDisplayNode (AccessibilityInternal)
 
+- (void)setAccessibilityElements:(NSArray *)accessibilityElements
+{
+  self.view.accessibilityElements = accessibilityElements;
```

Comment:
This method would cause the node to load its view if not yet loaded. Let's do something similar to the `isNodeLoaded` check in `-accessibilityElements` setter.